File: app/src/main/java/ru/gilgamesh/abon/motot/ui/profile/RecyclerViewImgGallery/RecyclerViewImgGallery.java
package ru.gilgamesh.abon.motot.ui.profile.RecyclerViewImgGallery;

import android.view.View;
import android.widget.ImageView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import ru.gilgamesh.abon.motot.R;

public class RecyclerViewImgGallery extends RecyclerView.ViewHolder {
    ImageView image;

    public RecyclerViewImgGallery(@NonNull View itemView) {
        super(itemView);
        this.image = itemView.findViewById(R.id.imageView);
    }
}

File: userprofile/src/main/java/ru/gilgamesh/abon/userprofile/presentation/imageGallery/RecyclerViewImgGallery.kt
package ru.gilgamesh.abon.userprofile.presentation.imageGallery

import android.view.View
import android.widget.ImageView
import androidx.recyclerview.widget.RecyclerView
import ru.gilgamesh.abon.userprofile.R

class RecyclerViewImgGallery(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var image: ImageView = itemView.findViewById(R.id.imageView)
}